src/OpenLoco/Vehicles/VehicleBody.cpp

```diff
@@ -268,62 +265,51 @@ namespace OpenLoco::Vehicles
         if (object_sprite_type == 0xFF)
             return;
 
-        auto distance_x = front_bogie->x - back_bogie->x;
-        auto distance_y = front_bogie->y - back_bogie->y;
-
-        auto offset = fastSquareRoot(distance_x * distance_x + distance_y * distance_y);
+        auto xDiff = front_bogie->x - back_bogie->x;
+        auto yDiff = front_bogie->y - back_bogie->y;
+        auto distanceBetweenBogies = Math::Vector::distance(Map::map_pos{ front_bogie->x, front_bogie->y }, Map::map_pos{ back_bogie->x, back_bogie->y });
```

Comment:
I think it would be clearer if these parameters were assigned to local variables.

Reply:
hmm perhaps if we had a method that returned a map_pos from the base struct.

Reply:
That would be even better. If you're planning something like that already in a follow-up, I'm fine with this as-is.